docs/development/core/server/kibana-plugin-core-server.savedobjectsexportbyobjectoptions.md

```diff
@@ -4,7 +4,7 @@
 
 ## SavedObjectsExportByObjectOptions interface
 
-Options for the [export by objects API](./kibana-plugin-core-server.savedobjectsexporter.exportbyobjects.md)
+Options for the 
```

Comment:
I properly adapted the docs on `SavedObjectsExportByObjectOptions` / `SavedObjectsExportByTypeOptions`, and yet, the `@link` is broken. 

If anyone see anything obvious I missed, please tell me. Else, given these generated docs will likely go away with packages anyway, it's probably fine.

Reply:
Funny: Running `yarn docs:acceptApiChanges --docs` again brought it back 🤷 

I pushed a commit with those changes